unitTests.php

```diff
@@ -40,4 +40,49 @@ public function testMalformedDisplayPlayer ()
         $case = displayPlayers($testInput1);
         
     }
-}
\ No newline at end of file
+}
+
+
+
+class Functions extends TestCase {
```

Comment:
All of the tests in here should be part of the same class, just add the methods into the one that sits above